spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResource.java

```diff
@@ -130,7 +127,7 @@ public boolean exists() {
 
 	@Override
 	public boolean isReadable() {
-		return !isBucket();
+		return !location.isBucket();
```

Comment:
just isBucket should work.

```suggestion
		return !isBucket();
```